Library/Homebrew/brew.rb

```diff
@@ -13,21 +13,14 @@
 $:.unshift(HOMEBREW_LIBRARY_PATH.to_s)
 require "global"
 require "tap"
+require "utils"
 
 if ARGV == %w[--version] || ARGV == %w[-v]
   puts "Homebrew #{HOMEBREW_VERSION}"
   puts "Homebrew/homebrew-core #{CoreTap.instance.version_string}"
   exit 0
 end
```

Comment:
Can you benchmark the performance before/after this change on `--version`? Thanks!